src/utils/dates.ts

```diff
@@ -20,4 +20,4 @@ export const daysUntil = (expirationDate: string | null) => {
   return !isNil(expirationDate)
     ? moment(expirationDate).diff(moment(), 'days')
     : undefined;
-}
```

Comment:
We should also add rule for an empty row at the end of file. This is not related to this PR.

Reply:
@dusan-kovacik-profiq can you add this next?

Reply:
Ok, as you have merged it, I will create a new one :)